FilePersistence/src/version_control/merge/ListMergeComponentV2.cpp

```diff
@@ -63,7 +63,9 @@ void ListMergeComponentV2::run(MergeData& mergeData)
 	{
 		auto map = computeAdjustedIndices(listId, mergeData);
 		mergeData.cg_.relabelChildrenUniquely(listId, map, mergeData.treeBase_.get());
-		// applyChanges(mergeData);
+		mergeData.cg_.updateListChanges(listId, map, mergeData.treeBase_.get());
```

Comment:
Also you should not call this function here, it should be a private function called from within, `relabelChildrenUniquely`